crypto/schnorr-adaptor-signature/keys.go

```diff
@@ -43,7 +43,8 @@ func NewDecyptionKeyFromBytes(decKeyBytes []byte) (*DecryptionKey, error) {
 	}
 
 	var decKeyScalar btcec.ModNScalar
-	decKeyScalar.SetByteSlice(decKeyBytes) //nolint:errcheck
+	// TODO: should we check the return value to identify whether overflow happened?
```

Comment:
Seems not needed here but it would be good for @SebastianElvis to verify

Reply:
Yep this is not needed. If we really want to check, then we can panick upon error (which means something is really wrong e.g., in the btcd lib)

Reply:
ok removed the todo